hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -1004,34 +1002,23 @@ onOpenChainIncrementTx openState newChainState newVersion depositTxId =
 --
 -- __Transition__: 'OpenState' → 'OpenState'
 onOpenChainDecrementTx ::
-  IsTx tx =>
   OpenState tx ->
   ChainStateType tx ->
   -- | New open state version
   SnapshotVersion ->
   -- | Outputs removed by the decrement
-  [TxOutType tx] ->
+  UTxOType tx ->
   Outcome tx
-onOpenChainDecrementTx openState newChainState newVersion distributedTxOuts =
-  -- Spec: if outputs(txω) = 𝑈ω
-  case decommitTx of
-    Nothing -> Error $ AssertionFailed "decrement observed but no decommit pending"
-    Just tx
-      | outputsOfTx tx == distributedTxOuts ->
-          -- Spec: txω ← ⊥
-          --       v  ← v
```

Comment:
note: should not we maintain these spec comments?

Reply:
No, these are also removed from the spec. There is a separate PR linked in the PR description.